modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -99,11 +148,12 @@ spec:
     interval: 30s
     relabelings:
     - action: replace
-      replacement: "<smcp_name>-<smcp_namespace>" # <2>
-      targetLabel: mesh_id
+      replacement: "basic-istio-system" # <2>
+      targetLabel: mesh_id # <3>
 ----
 <1> Since {product-title} monitoring ignores the `namespaceSelector` spec in `ServiceMonitor` and `PodMonitor` objects, you must apply the `PodMonitor` object in all mesh namespaces, including the control plane namespace.
-<2> A shared Prometheus instance can include metrics from multiple meshes. However, if you are deploying Kiali, the metrics must be associated with a single mesh. To associate metrics with a single mesh, add a unique `mesh_id` to each one. Doing so narrows the query scope in Kiali to only the relevant mesh metrics.
+<2> Since `"basic-istio-system"` is a combination of the SMCP name and namespace, you can replace it with any other unique value within a cluster with multiple SMCPs. You must replace `"basic-istio-system"` if you are using Kiali and have multiple SMCPs deployed in a cluster. You do not need to do this if you have a cluster-wide deployment. 
```

Comment:
1) "You must replace `"basic-istio-system"` if you are using Kiali and have multiple SMCPs deployed in a cluster." - Replace with what? I would prefer to just say that the value of `mesh_id` label must be unique for every mesh in the cluster.
2) "You do not need to do this if you have a cluster-wide deployment." - this sentence is very unclear. What don't I have to do? Changing "basic-istio-system" or adding it at all? I wanted to explain that `mesh_id` label is not necessary in cluster-wide deployments and then users don't need it in Pod/ServiceMonitor as well as in Kiali. In cluster-wide deployments users can remove this replacement:
```
    - action: replace
      replacement: "<smcp_name>-<smcp_namespace>" # <2>
      targetLabel: mesh_id
```
We can explain 2. in <3>.